### ISSUE TITLE
Running 2 clients

### ISSUE THREAD

**Issue by ozjohnson** [Serenalyw/MultiboxBot#1]
Hi @Serenalyw.  Firstly, this is a really cool project. I'm a beginning hobbyist programmer and have been using your code to learn as well as to hopefully work on my own bot for fun.  Made some progress, but am now stuck.

I'd like to run only two clients (leader + follower) however I'm lost on the maths and functions you've used for the screen configuration in the interface.py as it seems to be based on multiples of 5 (as I said: beginner :-/).  Do you have suggestions on how I can approach this?  I've tried a few things, but end up getting an "IndexError: list index out of range" after I launch the windows through the bot gui, which tells me whatever calculations I've done are either wrong, or I've missed something.  

Note: The gui launches the two windows, but doesn't resize them. When I click the "repair" button, it throws the above error.

I'd probably end up stripping down the code to only include the option for 1 and 2 clients as I won't be running more than that in my test env while I'm learning.  

Anyway, appreciate any help/advice you can provide!

Cheers/merci!

**Comment by SerenaTetart** [comment]
I'll add the 2 clients option tomorrow, the calculations are a bit weird I concede.

Don't forget to tell me if there are others problems or bugs.

**Comment by ozjohnson** [comment]
Thank you so much!  Will definitely let you know if I run into any other issues.

**Comment by ozjohnson** [comment]
Hi @Serenalyw.  Any chance of committing this soon-ish?  Was going to work on it over the weekend but blocked on launching the two clients.  No worries if not, of course--just keen to dive a bit more into how it functions.

**Comment by SerenaTetart** [comment]
Done, enjoy !

**Comment by ozjohnson** [comment]
Thanks for making these changes! It now allows for the credential population.  
A couple of observations (let me know if easier to open up separate issues or keep here?)
1) it seems that most code to send to groups or to check groups still uses %5.  This results in a IndexError: list index out of range (line 518, 578) when NBR_ACCOUNT == 2.  I notice that this is throughout the code and occurs when the characters login in.
2) When attempting to repair, line 395 throws an UnboundLocalError: local variable 'hwnd' referenced before assignment. I "fixed" this by simply copying the assignment locally, but not sure that's the right/best way to do it.  

Given the two errors above, I'm not able to get to the point of resizing the screens and actually trying to the bot itself.  Happy (and enjoy) seeing if I can get it to work, but thought I'd drop these findings in if you're interested in updating the code in a way that isn't completely hacked by a novice :D  Also, I enjoy QA as helps me learn.

**Comment by SerenaTetart** [comment]
Yes sorry, I added this too fast, the main idea of the algorithm was to make only multiples of 5; that's why we have *5 and %5 everywhere, I forgot about that...
The only way to add non multiples of 5 is to make special conditions for them.

However what you can do, is to launch 5 clients and delete 3 of them, it will work this way...

**Comment by Serenalyw** [comment]
This works for me:

![Capture d’écran 2022-11-20 212347](https://user-images.githubusercontent.com/65224852/202924427-ef7ed696-f85a-4b26-888e-c4a9c6f06db1.png)

**Comment by SerenaTetart** [comment]
This works for me:

![Capture d’écran 2022-11-20 212637](https://user-images.githubusercontent.com/65224852/202924507-d67e38b5-70f5-4a33-a30d-263e4138ef07.png)

**Comment by ozjohnson** [comment]
Thanks, I will try this!  I also found the reason why the screens weren't resizing I think:  I had two monitors connected to my gpu.  when I removed one and then started the game, it resized somewhat correctly (still some investigation on my part required).  When plugged the other display back in, it reverted to stacking the clients. 

Three other general questions if you don't mind:  if I load in as two lvl 1 paladins, the bot UI identifies both as Holy spec when logged in. If I immediately set the first Paladin to Protection, it throws this:
![image](https://user-images.githubusercontent.com/613644/202930109-67a0c4df-bc98-4b84-8d1f-2454f914e50c.png)
It doesn't seem like it's causing any issues with gameplay (that I can tell so far), as it does seem to be sending the right abilities to each, but thought I'd check.

The next question relates to auto-follow.  Does follow need to be set manually somehow?  I see the follow functions, but when I set my first Pally to Prot (Tank Spec), the second Pally (holy or ret) in the group won't autofollow, but will assist (attack when the target is tabbed).  So I don't think it's related to the error above, as at least some targeting functions are working.

Finally, is there a way to have the tank "click to move" attack, similar to the way the dps does?  Currently, when I tab to the next target, the dps clicks to move and attacks the target but the tank must be manually moved to attack range.  Not a big deal, but it would make for some dynamic continuous target attacking, as could set it to tab whenever there wasn't an existing target. No good for dungeons, but fun to play with the automation.

Anyway, just a few points.  Again, let me know if you'd like me to drop these questions somewhere else or if you're happy to review (or ignore! :D) them here.  

If you don't mind me asking, what is your expected "end state" for this project?  It's really cool and looking to see where you take it (looting, quest acceptance, etc?)

merci
